truss/templates/control_venv.Dockerfile.jinja

```diff
@@ -0,0 +1,12 @@
+{%- if requires_live_reload %}
+# This is hacky, but right now we pin the higher range of python version
```

Comment:
https://github.com/basetenlabs/truss/blob/e62fde1c833c91c77845263826f6d80d0b029605/pyproject.toml?plain=1#L74

To figure out later, but it's a little hard to untangle with this pinned [dependency](https://github.com/basetenlabs/truss/blob/e62fde1c833c91c77845263826f6d80d0b029605/truss/templates/control/requirements.txt?plain=1#L9) for now.